src/debug.c

```diff
@@ -664,22 +664,15 @@ NULL
         if ((o = objectCommandLookupOrReply(c,c->argv[2],shared.nokeyerr))
             == NULL) return;
 
-        int enabled = 0;
+        int light = 0;
         if (c->argc == 4) {
-            enabled = !strcasecmp(c->argv[3]->ptr,"1");
-            if(!enabled && strcasecmp(c->argv[3]->ptr,"1")) {
-                addReplyError(c,"only 1 or 0 is allowed");
-                return;
-            }
+            light = atoi(c->argv[3]->ptr);;
+            addReply(c,shared.ok);
```

Comment:
Is this line redundant?

Reply:
yeah, returning two replies on one command would make a huge mess.